modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comments.es.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React, {useCallback, useState} from 'react';
+
+import {createComment} from '../utils/client.es';
+import Comment from './Comment.es';
+
+export default ({commentChange, comments, entityId}) => {
+	const [comment, setComment] = useState('');
+	const [showNewComment, setShowNewComment] = useState(false);
+
+	const postComment = () => {
+		return createComment(comment, entityId).then(data => {
+			setComment('');
+			setShowNewComment(false);
+			commentChange([...comments, data]);
+		});
+	};
+
+	const unmountComment = useCallback(
```

Comment:
Yessss... I think it was because I can't use the same name for props and internal method... I'm using the _ notation for private methods now: https://github.com/boton/liferay-portal/pull/122/commits/33854ad927dbe6933144007f89f67f3221e5b740